src/utils/token_fiat_price.ts

```diff
@@ -0,0 +1,17 @@
+import type { FiatID, TokenID } from './fiat_oracle_types';
+
+/**
+ * Immutable data container class representing a market price in USD for a particular AR amount
```

Comment:
We'll support a wide range of fiat IDs. Lets be more generic in the documentation: `USD` --> `Fiat`

Reply:
4fc1b43c3640b7a0cfc720e016133f59b0996ff4